srcpkgs/fcitx-m17n/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'fcitx-m17n'
+pkgname=fcitx-m17n
+version=0.2.4
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="m17n-lib-devel fcitx-devel"
+depends="fcitx m17n-db"
```

Comment:
Does it need to depend on these?

Reply:
Well the dependency on `m17n-db` is there since M17n is not very useful if one doesn't also have the database, so might as well depend on it since the user would probably install it anyway. Besides, similar thing is done by `ibus-m17n` so it seemed like a good idea.

As for depending on `fcitx` explicitly, other packages that also add these sorts of IM engines to it also tend to do that, so that's what I'm basing it upon.

Should I remove those dependencies?

Reply:
I don't think so, I was just checking! Those sound like good justifications for it.

That said, any easy way to test this? Just install and launch?

Reply:
Yeah, testing shouldn't be too hard, assuming one already has fcitx set up on their computer. Just install and reload the daemon.

If one doesn't use fcitx, the way one would have to ensure that various applications use the proper input method would be to set the following environment flags as follows:

```sh
export GTK_IM_MODULE=fcitx
export QT_IM_MODULE=fcitx
export XMODIFIERS=@im=fcitx
```

After this, one could launch the daemon and an application to verify that this works.

At least on my (somewhat limited) testing it seems to work like it's supposed to.

Reply:
The dependency is correct. Normally, I work with all fcitx package, hopefully, I can find some time to experience with this one, soon.